free_fleet_cyclonedds/src/messages/MiddlewareMessages.h

```diff
@@ -0,0 +1,197 @@
+/****************************************************************
+
+  Generated by Eclipse Cyclone DDS IDL to C Translator
```

Comment:
I see that these messages are auto-generated. Is it possible to make that generation be part of the build process to make it easier to manage changes to the message definitions?